modules/virt-ssh-configuration-considerations.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+:_content-type: REFERENCE
+[id="virt-ssh-configuration-considerations_{context}"]
+= Access configuration considerations
+
+Each method for configuring access to a virtual machine (VM) has advantages and limitations, depending on the traffic load and client requirements.
+
+SSH key pair and virtctl::
+* Simple to configure.
+* Not recommended for high-traffic applications.
+* The API server must be able to handle the traffic load.
+* The clients must be able to access the API server.
+* The clients must have access credentials for the cluster.
+
+.Services
+
+Services provide excellent performance and are recommended for applications that are accessed from outside the cluster.
+
+Cluster IP service::
+* The internal cluster network must be able to handle the traffic load.
+* The clients must be able to access an internal cluster IP address.
+
+Node port service::
+* The internal cluster network must be able to handle the traffic load.
+* The clients must be able to access at least one node.
+
+Load balancer service::
+* If you use a MetalLB load balancer in layer 2 mode, a single network interface on each node must be able to handle the traffic load.
```

Comment:
This is not accurate. The node might have multiple node interfaces, and the service will be exposed on all of them. It will work regardless of the node interface the client is reaching the service from.

Reply:
this is for each and every service of type LB.
MetalLB will elect a node every time a L2 LB appears, so it can potentially be different (it's based on hashing, so it's not guaranteed)